### ISSUE TITLE
Minified JS broken with private class identifiers

### ISSUE THREAD

**Issue by PLUS-RobertEhrenleitner** [mrclay/jsmin-php#15]
I report this here once more after posting it on [autoptimize](https://github.com/futtta/autoptimize/issues/393).

When a class contains private attributes or methods, the necessary line break or semicolon is not inserted.
As a minimum non-working example, the following class in valid JavaScript syntax does not work:
````
class TestClass {
    #a = 3
    #b = 5
    #fnc1() {
        do_something()
    }
    #fnc(2) {
        do_anything()
    }
}
````
It is minified to:
````
class TestClass{#a=3#b=5#fnc1(){do_something()}#fnc(2){do_anything()}}
````
which contains several syntax errors: There should be line breaks or semicolons between all the attributes and methods.

*Assumption*: The hash sign is treated the same way like an operator, and lines beginning with an operator can be appended to the preceding line.

**Comment by PLUS-RobertEhrenleitner** [comment]
See [pull request #18](https://github.com/mrclay/jsmin-php/pull/18).

**Comment by PLUS-RobertEhrenleitner** [comment]
I added a test case.

**Comment by glensc** [comment]
forgot to check PR uses [issue close patterns](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

closing manually

**Comment by glensc** [comment]
thanks. released as 2.4.3
